core/src/main/java/hudson/Functions.java

```diff
@@ -643,6 +645,12 @@ public static void configureAutoRefresh(HttpServletRequest request, HttpServletR
         if (refresh) {
             response.addHeader("Refresh", SystemProperties.getString("hudson.Functions.autoRefreshSeconds", "10"));
         }
+
+        try {
+            ExtensionList.lookupSingleton(AutoRefresh.class).recordRequest(request, refresh);
+        } catch (Exception e) {
+            LOGGER.log(Level.FINE, "Failed to record auto refresh status in telemetry", e);
```

Comment:
```suggestion
            LOGGER.log(Level.WARNING, "Failed to record auto refresh status in telemetry", e);
```

since we do not expect problems here and would want to know about them.